### ISSUE TITLE
Connection jumps when reconnecting in DRD editor 

### ISSUE THREAD

**Issue by nikku** [camunda/camunda-modeler#1603]
__Describe the Bug__

When reconnecting a connection in the DRD editor I can see that the source anchor is temporarily forgotten. This results in a visible (and annoying I my point of view) jump during re-connection:

![capture hXsi1V_optimized](https://user-images.githubusercontent.com/58601/69530113-c9d32300-0f71-11ea-9df1-c4537cde0bd0.gif)

__Steps to Reproduce__

1. Connect two elements
2. Reconnect to different element
3. __Observe visible jump during hover / out__

__Expected Behavior__

* Reconnecting remembers source and target anchor (depending on which part the user drags) and preview does not show a jump.

__Environment__

 - OS: Any
 - Camunda Modeler Version: v3.5.0-dev

This behavior is not present in Camunda Modeler v3.4.1.

**Comment by philippfromme** [comment]
This is the same behavior as in bpmn-js. The noop preview is always drawn from the element's center.

![29-11-_2019_20-05-23](https://user-images.githubusercontent.com/7633572/69886717-a4cd1000-12e3-11ea-96ea-72d345a19202.gif)

Do we want to fix this in all of our editors?

**Comment by nikku** [comment]
> Do we want to fix this in all of our editors?

I'd do it.

**Comment by philippfromme** [comment]
Okay, thanks for clarification. 🙂

**Comment by nikku** [comment]
It is an issue to be fixed in the core, right?

**Comment by philippfromme** [comment]
Yes, it requires a fix in [ConnectionPreview](https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/connection-preview/ConnectionPreview.js#L149).

**Comment by nikku** [comment]
So it seems like that the current behavior has been decently well thought of. Marking this as an improvement and moving it back to backlog.
